app/__init__.py

```diff
@@ -13,27 +15,40 @@
     "default": "app.flask_config.DevelopmentConfig",
 }
 
-app = Flask(__name__)
 
-# initialize configuration values
-config_name = os.getenv("FLASK_ENV", 'default')
-app.logger.debug("Reading " + config_name + " configuration...")
+def create_app():
+    app = Flask(__name__)
 
-config_obj = CONFIG[config_name]
-app.logger.debug("config obj: " + config_obj)
-app.config.from_object(config_obj)
+    # initialize configuration values
+    config_name = os.getenv("FLASK_ENV", 'default')
+    app.logger.debug("Reading " + config_name + " configuration...")
 
-if app.config.from_pyfile('flask_config.py'):
-    app.logger.info("Successfully read instance configuration file.")
-app.logger.info("Flask config:")
-app.logger.info(app.config.items())
+    config_obj = CONFIG[config_name]
+    app.logger.debug("config obj: " + config_obj)
+    app.config.from_object(config_obj)
 
-# configure logging
-if not app.debug:
-    app.logger.setLevel(logging.INFO)
+    if app.config.from_pyfile('flask_config.py'):
+        app.logger.info("Successfully read instance configuration file.")
+    app.logger.info("Flask config:")
+    app.logger.info(app.config)
 
-os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = (
-    "/home/samschaevitz/Downloads/BERT Classification-9a8b5ef88627.json")
+    # configure logging
+    if not app.debug:
+        app.logger.setLevel(logging.INFO)
 
-from app import routes  # nopep8
-from app import task_routes  # nopep8
+    os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = (
+        "/home/samschaevitz/Downloads/BERT Classification-9a8b5ef88627.json")
```

Comment:
This path doesn't look like it should be submitted - is this related to .credz?

Reply:
yep; fixed (is by default a relative path from the flask config